tests/fields/fields.py

```diff
@@ -932,6 +938,7 @@ class BlogPost(Document):
             tags = ListField(StringField())
             authors = ListField(ReferenceField(User))
             generic = ListField(GenericReferenceField())
+            access_list = ListField(required=False, default=[], choices=AccessLevelChoices, display_sep=u',')
```

Comment:
We shouldn't need `required=False` and `default=[]`. And, if I understand correctly, `display_sep` is a new kwarg we're introducing in this PR, hence we should document it. The docstring for the `ListField` might be the best place. You could also add it as an explicit kwarg to `ListField.__init__` for better discoverability.

Finally, `', '` instead of `u','` would be a tad nicer.